framework/PageDashboard/PageDashboardCountBar.tsx

```diff
@@ -1,49 +1,157 @@
-import { CardBody, Flex, FlexItem, Title } from '@patternfly/react-core';
-import { ArrowRightIcon } from '@patternfly/react-icons';
+import { ChartDonut, ChartLabel, ChartLabelProps } from '@patternfly/react-charts';
+import { CardBody, Title } from '@patternfly/react-core';
+import { CSSProperties } from 'react';
 import { Link } from 'react-router-dom';
 import { PageDashboardCard } from './PageDashboardCard';
 
 export type PageDashboardCountBarProps = {
   counts: {
     title: string;
-    count: number;
+    total?: number;
     to: string;
+    counts?: { label: string; count: number; color: string; link?: string }[];
   }[];
 };
 
 export function PageDashboardCountBar(props: PageDashboardCountBarProps) {
   return (
     <PageDashboardCard width="xxl" isCompact>
       <CardBody>
-        <Flex
-          spaceItems={{ default: 'spaceItems2xl' }}
-          style={{ rowGap: 16 }}
-          justifyContent={{ default: 'justifyContentSpaceEvenly' }}
+        <div
+          style={{
+            display: 'flex',
+            flexWrap: 'wrap',
+            justifyContent: 'space-evenly',
+            paddingLeft: 8,
+            paddingRight: 8,
+            gap: 16,
+          }}
         >
-          {props.counts.map((item) => (
-            <FlexItem key={item.title}>
-              <Flex
-                alignItems={{ default: 'alignItemsCenter' }}
-                spaceItems={{ default: 'spaceItemsMd' }}
-              >
-                <FlexItem>
-                  <Title headingLevel="h3" size="xl">
+          {props.counts.map((item, index) => {
+            const id = item.title.toLowerCase().replace(/ /g, '-');
+            const total: number =
+              item.total ??
+              item.counts?.reduce<number>((acc: number, curr) => acc + (curr.count ?? 0), 0) ??
+              0;
+            return (
+              <div id={id} key={index} style={{ display: 'flex', gap: 12, alignItems: 'center' }}>
+                <Link to={item.to} style={{ color: 'var(--pf-global--text--Color)' }}>
+                  <Title
+                    headingLevel="h3"
+                    size="xl"
+                    style={{ whiteSpace: 'nowrap', textDecoration: 'none' }}
+                  >
                     {item.title}
                   </Title>
-                </FlexItem>
-                <FlexItem>
-                  <span style={{ fontSize: 'xx-large', lineHeight: 1 }}>{item.count}</span>
-                </FlexItem>
-                <FlexItem>
-                  <Link to={item.to}>
-                    <ArrowRightIcon />
-                  </Link>
-                </FlexItem>
-              </Flex>
-            </FlexItem>
-          ))}
-        </Flex>
+                </Link>
+                {total && 'counts' in item && item.counts ? (
```

Comment:
The last two arguments of this conditional is confusing to parse -  could you explain what this is checking?

Reply:
comments added to explain